CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
   `KVM_GET_MSR_FEATURE_INDEX_LIST` and `KVM_GET_MSRS` system ioctls.
 - [[#221](https://github.com/rust-vmm/kvm-ioctls/pull/221)] Add
   `Cap::ArmPmuV3`.
+- [[#223](https://github.com/rust-vmm/kvm-ioctls/pull/223)] feat(aarch64):
+  generic `get/set_one_reg`
```

Comment:
Can you be more explicit with the change? What is important for customers reading this is not that get/set is generic, but what is the functionality that it now supports: i.e. support for getting and setting registers that are not of a fixed size.

Reply:
Updated.